Makefile

```diff
@@ -67,19 +67,25 @@ help: ## This help.
 # -- Build --
 
 .PHONY: build
-build: get $(BINDIR)/$(BINNAME) ## Build go code
+build: get $(BINDIR)/$(BINNAME) # Build go code
```

Comment:
We need ## to show up as help, when just typing 'make'

Reply:
I was wondering why we had that. done